src/org/omegat/filters2/master/FilterMaster.java

```diff
@@ -202,6 +203,8 @@ public IFilter loadFile(String filename, FilterContext fc, IParseCallback parseC
             filterObject = lookup.filterObject;
 
             filterObject.parseFile(inFile, lookup.config, fc, parseCallback);
+        } catch (TranslationException e) {
+            throw e;
```

Comment:
Here is a core part of the change.
When filter got error, TranslationException will deliver to parent caller.